package.json

```diff
@@ -33,8 +33,8 @@
     "postinstall": "husky install",
     "lint": "eslint --ext .js,.ts .",
     "lint-fix": "eslint --fix --ext .js,.ts .",
-    "lint-prettier": "prettier '**/*.{yml,yaml,md}' --write",
-    "lint-prettier-ci": "prettier '**/*.{yml,yaml,md}' --check",
+    "lint-prettier": "prettier \"**/*.{yml,yaml,md}\" --write",
```

Comment:
I think these 2 should be reverted?

Reply:
These changes were necessary to run prettier on windows (with yarn 3). On the command line I can:

```
yarn prettier '**/*.{yml,yaml,md}' --write
```

But for some reason the commit hook always fails on this line with the previous code, but works with the change. I figure others could have the same problem, which is why I included this change. If you'd like it reverted, I'm happy to, but I will have to commit with `--no-verify`.

Reply:
hmm this is odd. If so, I think it's good to have.